soapui/pom.xml

```diff
@@ -203,13 +203,13 @@
         <dependency>
             <groupId>com.smartbear.utils.analytics</groupId>
             <artifactId>analytics-core</artifactId>
-            <version>5.5.0-SNAPSHOT</version>
+            <version>5.5.0-AT</version>
         </dependency>
 
         <dependency>
             <groupId>com.smartbear.utils.analytics</groupId>
             <artifactId>out-app-analytics-provider</artifactId>
-            <version>5.5.0-SNAPSHOT</version>
+            <version>5.5.0-AT</version>
```

Comment:
it looks like the PR is broken, current "next" branch is based on 5.5.0 version. These 2 should not be changed.